src/shipit_code_coverage/tests/test_artifacts.py

```diff
@@ -59,3 +60,25 @@ def add_dir(files):
 
     with pytest.raises(Exception, message='suite and chunk can\'t both have a value'):
         a.get(chunk='xpcshell-7', suite='mochitest')
+
+
+@mock.patch('shipit_code_coverage.taskcluster.get_task_artifacts')
+@mock.patch('shipit_code_coverage.taskcluster.download_artifact')
+def test_download(mocked_download_artifact, mocked_get_task, LINUX_TEST_TASK_ARTIFACTS):
+    a = ArtifactsHandler([], [], parent_dir='')
+    mocked_get_task.return_value = LINUX_TEST_TASK_ARTIFACTS['artifacts']
+
+    a.download(LINUX_TEST_TASK_ARTIFACTS)
```

Comment:
The argument of download should be a task definition, rather than the artifacts reply.

Reply:
Like one of the elements from the `task-group_1.json` file in fixtures/, for example the one with name `test-linux64-ccov/opt-mochitest-devtools-chrome-e10s-4`. And then use the linked artifacts (https://queue.taskcluster.net/v1/task/AMhPZboeTSa_lax-wl36IA/artifacts).